docs/static/releasenotes.asciidoc

```diff
@@ -15,6 +16,11 @@ This section summarizes the changes in the following releases:
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-11]]
+=== Logstash 5.6.11 Release Notes
+* Fix ArrayIndexOutOfBoundsException from empty string field reference #9881
```

Comment:
I suggest either not including the PR number or using an url syntax element so there's a proper hyperlink